Panic error - DeliveryScanQueryByID, ScanFromBlock function

In `DeliveryScanQueryByID` (`token/services/network/fabric/lookup/deliveryqs.go`), Delivery is not set and can lead to a panic error.

Solution is simply to setup Delivery value in `DeliveryScanQueryByID` at `token/services/network/fabric/lookup/deliveryllm.go:129`

I'll handle the fix right away

```
2025-09-24 13:09:37.258 UTC [token-sdk.network.fabric.llm] func2 -> WARN 054 listeners for TXs [[tamhtlc.cpi32316633386239623738353461343063336432333235396262313665393534313838626338633339653231373236346265643930306233323166376130363738]] timed out. Last Block Num [7289]. Either the TX finality is too slow or it reached finality too long ago and were evicted from the events cache. The IDs will be queried directly from ledger (when the batch is cut)...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x283d8be]

goroutine 27857 [running]:
github.com/hyperledger-labs/fabric-smart-client/platform/fabric.(*Delivery).ScanFromBlock(...)
	/go/src/github.com/banquedefrance/dl3s/backend/dl3s/vendor/github.com/hyperledger-labs/fabric-smart-client/platform/fabric/delivery.go:55
github.com/hyperledger-labs/fabric-token-sdk/token/services/network/fabric/lookup.(*DeliveryScanQueryByID).queryByID(0xc02da6d080, {0x3d5b5e8, 0x670d5a0}, {0xc05a5e6560, 0x1, 0xc02c25f7d0?}, 0xc0002b10a0, 0x1c79, 0xc02c313140)
	/go/src/github.com/banquedefrance/dl3s/backend/dl3s/vendor/github.com/hyperledger-labs/fabric-token-sdk/token/services/network/fabric/lookup/deliveryqs.go:117 +0xd5e
created by github.com/hyperledger-labs/fabric-token-sdk/token/services/network/fabric/lookup.(*DeliveryScanQueryByID).QueryByID in goroutine 27856
	/go/src/github.com/banquedefrance/dl3s/backend/dl3s/vendor/github.com/hyperledger-labs/fabric-token-sdk/token/services/network/fabric/lookup/deliveryqs.go:40 +0x1f8
```

--- a/token/services/network/fabric/lookup/deliveryllm.go
+++ b/token/services/network/fabric/lookup/deliveryllm.go
@@ -127,7 +127,8 @@ func (p *deliveryBasedLLMProvider) NewManager(network, channel string) (Listener
 			Logger:   logger,
 		},
 		&DeliveryScanQueryByID{
-			Channel: ch,
+			Delivery: ch.Delivery(),
+			Channel:  ch,
 		},
 		p.tracerProvider.Tracer("finality_listener_manager", tracing.WithMetricsOpts(tracing.MetricsOpts{})),
 		p.newMapper(network, channel),

